eq.yml

```diff
@@ -72,8 +72,11 @@ services:
       REACT_APP_BASE_NAME: /eq-author
       REACT_APP_API_URL: http://localhost:4000/graphql
       REACT_APP_USE_MOCK_API: "false"
+      REACT_APP_PUBLISHER_URL: http://localhost:9000/publish
```

Comment:
this should be `http://eq-publisher:9000/publish` as the communication is within the docker network